src/stories/UI/Toast.stories.mdx

```diff
@@ -0,0 +1,343 @@
+import { Meta, Canvas, Story, Preview, Props } from "@storybook/addon-docs";
+import { Toast } from "../../indexLib";
+import { ArgsTable, Source } from "@storybook/addon-docs";
+import dedent from "ts-dedent";
+import {
+  ToastProviderDefault,
+  ToastProviderPriority,
+  ToastProviderActionLabel,
+  ToastProviderAutoDismiss,
+  ToastProviderCloseOnAction,
+  ToastProviderWithoutIcon,
+} from "./Toast.examples";
+
+<Meta title="Navigation/Toast" component={Toast} />
```

Comment:
'Status' I think is more fitting as it doesn't really feel like Navigation. This is consistent with [Adobe](https://react-spectrum.adobe.com/react-spectrum/Toast.html) as well. 
<img width="234" alt="Screenshot 2023-05-26 at 11 23 46" src="https://github.com/action-is-hope/shelley/assets/678624/ffbf5d79-6519-456d-bfc3-ae5fca257364">

Reply:
I could swear they just moved it from navigation. I will update.

Reply:
Done. https://github.com/action-is-hope/shelley/pull/99/commits/eae289e3f466926481eceda6c70ebb45d30a8c68